bugbug/similarity.py

```diff
@@ -106,6 +111,21 @@ def get_text(self, bug, all_comments=False):
 
         return "{} {}".format(bug["summary"], comments)
 
+    def get_similar_bugs(self, bug):
+        similar_bug_ids = self.get_similar_bugs_child(bug)
+        similar_bugs = bugzilla.get(similar_bug_ids)
+        if self.end_to_end:
+            duplicatemodel = DuplicateModel.load("duplicatemodel")
```

Comment:
How fast is this? If it's slow, maybe we should load it just once in the constructor.

Reply:
Yeah that's probably more efficient.